### ISSUE TITLE
Keyboard has no focus

### ISSUE THREAD

**Issue by bmihaila** [psifidotos/workflow-project#77]
Hi

So now hopefully the last bug for today ;).
Sometimes when activating the plasmoid it does not have the keyboard focus, which means using the arrow keys does not work and if I type any key it gets directed to the KDE runner. I cannot now say exactly what I need to do to lose focus but when this happens I can only get the keyboard focus back by e.g. clicking the "Edit the workarea name" and from then on all the keys are again sent to the plasmoid.

**Comment by psifidotos** [comment]
I know, it happens to me also some times but I havent managed to reproduce it... In some of these cases something doesnt go well with the Ctrl, for example using the wheel alone it zooms in and out... It is like Ctrl has been kept pressed but I am not sure...

If you hide pop up and show it again, doesnt it regain focus? if not check the behavior with the scroll wheel to tell me if that's the case...

**Comment by bmihaila** [comment]
> If you hide pop up and show it again, doesnt it regain focus?

No, that does not bring focus back. The most reliable method was really to click the rename workspace field.

> if not check the behavior with the scroll wheel to tell me if that's the case...

I think I had this earlier, too but now it does not seem to be a problem anymore, but it might be related with how to reproduce ...

So, I managed to reproduce at least in one case where sometimes focus is back after hiding+reopen the plasmoid and sometimes not. 
Open the plasmoid with keyboard focus working and then while it is open, use the global shortcut to change activities. This does not close the plasmoid but now keyboard focus is gone. Like I said, sometimes closing and reopening does not bring it back but sometimes yes.
